db/controllers/controllers.js

```diff
@@ -1,12 +1,13 @@
 const { user } = require("pg/lib/defaults");
-
 const {
   fetchTopics,
   fetchArticlesById,
   fetchArticles,
   fetchCommentsById,
   createComment,
+  updateVotes,
 } = require("../models/models");
+const { convertTimestampToDate } = require("../seeds/utils");
```

Comment:
No need to bring this in - it's a utility function that we've created for seeding purposes.